src/pages/hcp/api-docs/vault-secrets/[[...page]].tsx

```diff
@@ -118,23 +122,28 @@ export const getStaticPaths: GetStaticPaths<ApiDocsParams> = async (ctx) => {
 export const getStaticProps: GetStaticProps<
 	ApiDocsViewProps | OpenApiDocsViewProps,
 	ApiDocsParams
-> = async ({ params }: { params: ApiDocsParams }) => {
+> = async ({ params }: GetStaticPropsContext<ApiDocsParams>) => {
+	// Fetch all version data, based on remote `stable` & `preview` subfolders
+	const versionData = await fetchCloudApiVersionData(GITHUB_SOURCE_DIRECTORY)
```

Comment:
Switched to fetch version data before the "is the new template flag on" conditional, since I think we'll just be re-using this `fetchCloudApiVersionData` code for now (until we start to tackle 🎟️  [[discuss] Plan out content API for OpenAPI specs used on Dev Dot to avoid loading from GitHub directly](https://app.asana.com/0/1100423001970639/1205041193652635/f)).